### PR TITLE
added missing test (returns an empty array when requesting comments f…

### PR DESCRIPTION
**Pull request by kapilamin**
…rom a valid article that doesn't have any associated comments) for task 6 get all comments for an article.'

### CODE REVIEW

File: beta files/comments.controllers.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const selectArticleById  = require("../models/articles.models");
+const { insertCommentsByArticleId, selectCommentsByArticleId } = require("../models/comments.models");
+
+exports.getCommentsByArticleId = (req, res, next) => {
+    const { article_id } = req.params;
+    selectArticleById(article_id) 
+      .then(article => {
+        if (!article) {
```
Comment (nc-hannah-burks) [inline beta files/comments.controllers.js:8]: We shouldn't need this as selectArticleById already throws the appropriate error if it returns an empty row 🙂 instead we can add select selectArticleById(article_id) and selectCommentsByArticleId in a promise.all and it will handle the error if given a non existant article ID

**Review by nc-hannah-burks — COMMENTED** [review]
Almost there Kapil - but we can refactor slightly to simplify our logic 😊

File: beta files/comments.controllers.js (hunk 1 above)
Comment (kapilamin) [reply beta files/comments.controllers.js:8]: Think I've fixed it 😊
